js/app/components/AddressAutosuggest.js

```diff
@@ -384,9 +382,12 @@ class AddressAutosuggest extends Component {
     this.placeholder = localize('placeholder', adapter, this)
     this.poweredBy = localize('poweredBy', adapter, this)
     this.theme = localize('theme', adapter, this)
-    this.highlightFirstSuggestion = localize('highlightFirstSuggestion', adapter, this)
```

Comment:
Does this break anything? If not, I'd keep it just in case

Reply:
there is a warning in the console (you can see it on master or on demon instance), which say that it should be a boolean, so i modified it ( see here https://github.com/moroshko/react-autosuggest )

Reply:
actually it might break something for GB address picker so I changed it see commit 376834933dc67e2b9e96480cd761dfdc4eae16d8